app/Main.hs

```diff
@@ -80,7 +78,8 @@ graphOptions = GraphOptions
         command "rank" (info (pure Rankings) (progDesc "Show the most depended on modules")),
         command "longest" (info (pure FindLongest) (progDesc "Show the longest shortest path between two modules")),
         command "select" (info selectCmd (progDesc "Select a subset of the graph from a starting module")),
-        command "to-csv" (info csvCmd (progDesc "Convert to a CSV of edges compatible with SQLite and Gephi"))
+        command "to-csv" (info csvCmd (progDesc "Convert to a CSV of edges compatible with SQLite and Gephi")),
+        command "cat" (info catCmd (progDesc "Concatenate multiple graphs"))
```

Comment:
I kind of don't like that this still requires the -g even though it doesn't use it.

Reply:
Yeah I think `-g` being shared for all of `graph` isn't quite right. I think maybe it could be a reusable option parser that some of these commands use. Same with `-r`